Add new page Resetpassword.tsx

fenglin9686 opened the pull request:
add new page call ResetPassword.tsx for reset user password in login folder, haven't connected to backend yet

Fenglin2968 committed:
add new page call ResetPassword.tsx for reset user password in login foloder, havent connect to backend yet

Fenglin2968 committed:
Finished Merge

AnthonyNGarcia commented on src/Components/App.tsx:
```diff
@@ -33,7 +34,8 @@ function App():ReactElement{
       {/* {token ? <Drawer /> : null} */}
       {isLoggedIn ? <Drawer /> : null}
       <Switch>
-        <Route path="/" component={LoginPage} exact />
+        {/* <Route path="/" component={LoginPage} exact /> */}
+        <Route path="/" component={ResetPassword} exact />
```
Let's not override the "/" path. Make a new route, maybe called "/reset" or something. Right now it looks like the application will no longer show the login page.

AnthonyNGarcia committed:
Fixed routes from testing purposes back to normal

-Uncommented the "/" route path for login, thus preserving the normal runtime/integrity of the application
-Renamed the path for the ResetPassword component to "/reset" in order to avoid a naming conflict with the login route

Commit:
Anthony's commit: fixed unused imports causing compile failure

kdgreen398 reviewed (changes requested):
In ResetPassword.txt, the TextFields and Button should be wrapped inside of a Form tag

Fenglin2968 committed:
Change label in second input field to comfirm password and add two input field under the form

Fenglin2968 committed:
Change label from comfirm password to confirm password and change form validatation for empty input instead show the error message
